Domain Connect Spec Draft.adoc

```diff
@@ -1952,6 +1952,50 @@ record of "xyz.bar" (or "xyz.bar.example.com." in bind format).
 A value of @ for host in the template is a placeholder for an empty value. In other words @
 would point to "bar.example.com." when the same template is applied to domain=example.com and host=bar.
 
+In summary.
+
+....
+       +------+
+       | user |
+       +------+
+          |
+          v
+   +----------------------------+
+   | https://.../apply?host=xyz |
+   +----------------------------+
+          |
+          v
+   +----------------------------------------+
+   | get domain from template: example.com. |

```

Comment:
What is the meaning of "get domain from template"? Or is it the domain of apply call (L1964) ?